internal/server/errors.go

```diff
@@ -19,7 +19,14 @@ var (
 	errFailedToCreateChannelData              = errors.New("failed to create channel data from packet")
 	errRelayAlreadyAllocatedForFiveTuple      = errors.New("relay already allocated for 5-TUPLE")
 	errUnsupportedTransportProtocol           = errors.New("RequestedTransport must be UDP or TCP")
+	errUnsupportedClientProtocol              = errors.New("cannot allocate tcp if client connection transport is not TCP/TLS")
```

Comment:
Please change lowercase `tcp` to `TCP`

Reply:
Also, I prefer to use the following wording for errors: `failed to ...`. E.g. in this instance `failed to allocate TCP...`